src/histolab/slide.py

```diff
@@ -455,6 +431,26 @@ def _thumbnail_size(self) -> Tuple:
             ]
         )
 
+    def _valid_coordinates(self, coords: CoordinatePair) -> bool:
```

Comment:
I'm not sure removing the verb helps in conveying the right message: to me, here it seems you are getting new valid coordinates, instead of validating them

Reply:
ok i propose something like `has_valid_coords` because it's a method that lives in a class so it's referring to an object and it's not a generic function applicable to anything

Reply:
yep because the coordinates must be valid for that particular slide, ok for `has_valid_coords`

Reply:
done